chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -148,6 +160,41 @@ private User create(final CreateUserFromInvitationRequest userInfo)
         return result;
     }
 
+    private Set<String> validateCreateUserFromInvitationRequest(final CreateUserFromInvitationRequest request) {
+        Set<String> validationErrors = new HashSet<String>();
+
+        if (request.getUser().getSubjectName().length() > getMaxLength("subjectName")) {
+            validationErrors.add(getMaxLengthErrorMessage("user.subjectName.maxlength", "subjectName"));
+        }
+        if (request.getUser().getFullName().length() > getMaxLength("fullName")) {
+            validationErrors.add(getMaxLengthErrorMessage("user.fullName.maxlength", "fullName"));
+        }
+        if (request.getUser().getFriendlyName().length() > getMaxLength("friendlyName")) {
+            validationErrors.add(getMaxLengthErrorMessage("user.friendlyName.maxlength", "friendlyName"));
+        }
+        if (request.getUser().getTitle().length() > getMaxLength("title")) {
+            validationErrors.add(getMaxLengthErrorMessage("user.title.maxlength", "title"));
+        }
+        if (request.getUser().getEmail().length() > getMaxLength("email")) {
+            validationErrors.add(getMaxLengthErrorMessage("user.email.maxlength", "email"));
+        }
+        if (request.getUser().getPhoneNumber().length() > getMaxLength("phoneNumber")) {
+            validationErrors.add(getMaxLengthErrorMessage("user.phoneNumber.maxlength", "phoneNumber"));
+        }
+        return validationErrors;
+    }
+
+    private Integer getMaxLength(final String field) {
+        return Integer.parseInt(String.format(
+                messageSource.getMessage(new DefaultMessageSourceResolvable("maxLength." + field),
+                        LocaleContextHolder.getLocale())));
+    }
+
+    private String getMaxLengthErrorMessage(final String errorKey, final String field) {
+        return String.format(messageSource.getMessage(new DefaultMessageSourceResolvable(errorKey),
```

Comment:
I wrote ErrorMessageUtil as a cleaner way to get error messages, can you start using that?

Reply:
Fixed